.goreleaser.yml

```diff
@@ -0,0 +1,140 @@
+project_name: timestamp-authority
+
+env:
+  - GO111MODULE=on
+  - CGO_ENABLED=0
+  - DOCKER_CLI_EXPERIMENTAL=enabled
+  - COSIGN_EXPERIMENTAL=true
+
+# Prevents parallel builds from stepping on eachothers toes downloading modules
+before:
+  hooks:
+    - go mod tidy
+    - /bin/bash -c 'if [ -n "$(git --no-pager diff --exit-code go.mod go.sum)" ]; then exit 1; fi'
+# if running a release we will generate the images in this step
+# if running in the CI the CI env va is set by github action runner and we dont run the ko steps
+# this is needed because we are generating files that goreleaser was not aware to push to GH project release
+    - /bin/bash -c 'if [ -n "$CI" ]; then make sign-container-release; fi'
+
+# TODO: uncomment when if move to sigstore or point to haydentherapper the go modules
+# gomod:
+#   proxy: true
+
+sboms:
+- artifacts: binary
+
+builds:
+  - id: timestamp-server
+    binary: timestamp-server-linux-{{ .Arch }}
+    no_unique_dist_dir: true
+    main: ./cmd/timestamp-server
+    goos:
+      - linux
+    goarch:
+      - amd64
+      - arm64
+      - arm
+      - s390x
+      - ppc64le
+    goarm:
+      - 7
+    flags:
+      - -trimpath
+    mod_timestamp: '{{ .CommitTimestamp }}'
+    ldflags:
+      - "{{ .Env.LDFLAGS }}"
+
+  - id: timestamp-cli
+    binary: timestamp-cli-{{ .Os }}-{{ .Arch }}
+    no_unique_dist_dir: true
+    main: ./cmd/timestamp-cli
+    goos:
+      - linux
+      - darwin
+      - windows
+    goarch:
+      - amd64
+      - arm64
+      - arm
+      - s390x
+      - ppc64le
+    goarm:
+      - 7
+    ignore:
+      - goos: windows
+        goarch: arm64
+      - goos: windows
+        goarch: arm
+      - goos: windows
+        goarch: s390x
+      - goos: windows
+        goarch: ppc64le
+    flags:
+      - -trimpath
+    mod_timestamp: '{{ .CommitTimestamp }}'
+    ldflags:
+     - "{{ .Env.LDFLAGS }}"
+
+  - id: fetch-tsa-certs
```

Comment:
We don't need to create a separate binary for this, this will only be used by production maintainers to fetch a cert once or on expiration. Running this with `go run` should be fine.

Reply:
ok, i was unsure of that, will remove